tests/test_wheel.py

```diff
@@ -11,19 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import os
+from zipfile import ZipFile
+
 import pytest
+from tests.conftest import TESTS_DIR
```

Comment:
I appreciate the intention, but importing from `tests.confest` feels weird to me, since I believe its primary purpose is for defining shared fixtures and plugins, which don't need to be imported:

https://docs.pytest.org/en/latest/fixture.html#conftest-py-sharing-fixture-functions
https://docs.pytest.org/en/latest/writing_plugins.html#conftest-py-plugins

I think it's fine to only define `TESTS_DIR` in this file for now; we can extract it out later (maybe to `helpers`) if necessary.

Reply:
Fixed in latest commit, though we have places like https://github.com/pypa/twine/blob/master/tests/test_upload.py#L24 and https://github.com/pypa/twine/blob/master/tests/test_package.py#L24 where we are using the absolute names instead of relative.

Maybe a single PR moving some of the common functionalities to `tests/helper.py` can be done to address this, and perhaps other functions I can find repeated among the tests

Reply:
Good point. I like the idea of test refactoring PRs, though I'd suggest starting with small scope, to make them easier to review and merge.